pages/[id].tsx

```diff
@@ -264,25 +266,46 @@ const CandyMachine: NextPage = () => {
                                             )}
                                         </div>
                                     </div>
-                                    <div className='mt-5'>
-                                        <h4>Unminted NFTs - {itemsRemaining}</h4>
-                                        <div className='mt-3 nfts-grid'>
-                                            {new Array(5).fill(0).map((_, index) => (
-                                                <NftCard
-                                                    title={`CryptoDude #${index}`}
-                                                    key={index}
-                                                    imageLink={'/favicon.ico'}
-                                                    buttons={[
-                                                        {
-                                                            text: 'View in Solscan',
-                                                            as: 'link',
-                                                            variant: 'invisible',
-                                                            hash: '14eoYMYLY19gtfE1gwWDhnjDD3fDjGTQTGyicBKT33Ns',
-                                                        },
-                                                    ]}
-                                                />
-                                            ))}
-                                        </div>
+                                    <div className='mt-7'>
+                                        {!nfts.length ? (
+                                            <>
+                                                <h4>NFTs Preview · {itemsRemaining}</h4>
+                                                <Text as='p' className='mt-3 mb-4'>
+                                                    Upload cache file to preview NFTs
+                                                </Text>
+                                            </>
+                                        ) : (
+                                            <>
+                                                <h4>Unminted NFTs - {nfts.length}</h4>
```

Comment:
I think that this value can be obtained from the same way as in the nfts header
`${itemsRemaining + mintedNfts.length}`